swc-main.py

```diff
@@ -0,0 +1,108 @@
+import requirements_check
+from requirements_check import InvalidCheck, CHECKER
+import api as API
+import optparse as _optparse
+import sys as _sys
+
+HOST = "installation.software-carpentry.org"
+
+# Comment out any entries you don't need
+CHECKS = [
+# Shell
+    'virtual-shell',
+# Editors
+    'virtual-editor',
+# Browsers
+    'virtual-browser',
+# Version control
+    'git',
+    'hg',              # Command line tool
+    #'mercurial',       # Python package
+    'EasyMercurial',
+# Build tools and packaging
+    'make',
+    'virtual-pypi-installer',
+    'setuptools',
+    #'xcode',
+# Testing
+    'nosetests',       # Command line tool
+    'nose',            # Python package
+    'py.test',         # Command line tool
+    'pytest',          # Python package
+# SQL
+    'sqlite3',         # Command line tool
+    'sqlite3-python',  # Python package
+# Python
+    'python',
+    'ipython',         # Command line tool
+    'IPython',         # Python package
+    'argparse',        # Useful for utility scripts
+    'numpy',
+    'scipy',
+    'matplotlib',
+    'pandas',
+    # 'sympy',
+    # 'Cython',
+    # 'networkx',
+    # 'mayavi.mlab',
+    ]
+
+def python_version_check():
+    print("Checking for python version...")
+    if _sys.version_info < (2, 6):
+        print('check for Python version (python):')
+        print('outdated version of Python: ' + _sys.version)
+        return False
+    return True
+
+if __name__ == '__main__':
+
+    if python_version_check():
+        print('Passed')
+    else:
+        print('Failed')
+        print('Install a current version of Python!')
+        print('https://www.python.org/downloads/release/python-352/')
```

Comment:
I'd change that to `Install the newest version from https://www.python.org/downloads/ or ask instructors for help`.